index.html

```diff
@@ -0,0 +1,130 @@
+        <li>/</li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </nav>
+    <section class="home-intro">
+      <img src="images/home-intro.jpg" alt="man hugs smiley golden retriever" width="200">
+      <h1>Adopt A Dog</h1>
+      <p>Bubble tape turtlenecks blur zebra stripes netscape navigator, britpop how do you like them apples toyota previa apollo 13 slacker millenials. Saved by the bell carlton dance when you’re lost out there and you’re all alone joey tribbiani. Hot pockets skidz and zubas honda accord dotcom bubble courtney love, parting your hair down the middle.</p>
+    </section>
+    <aside>
+      <h2>From Our Blog</h2>
+      <article>
+        <img src="images/blog-1.jpg" alt="man wraps his arm around his black dog while sitting in a canyon" width="200">
+        <h3>Traveling With Your Dog</h3>
+        <p>T-shirts space jam sideburns boy meets world puff daddy encarta.</p>
+        <a href="blog.html"><p>Read More >></p></a>
+      </article>
+      <article>
+        <img src="images/blog-2.jpg" alt="four tall dogs of varying breeds sit at attention facing the individual who is holding their leashes" width="200">
+        <h3>How to Walk Multiple Dogs</h3>
+        <p>Nirvana seattle supersonics stonewashed blue jeans harrison ford.</p>
+        <a href="blog.html"><p>Read More >></p></a>
+      </article>
+      <article>
+        <img src="images/blog-3.jpg" alt="little girl holds up a ball in front of her sitting dog" width="200">
+        <h3>Teach Your Dog to Fetch!</h3>
+        <p>Skate tees hi-top fade, snapple hot pink zebra stripes troll dolls.</p>
+        <a href="blog.html"><p>Read More >></p></a>
+      </article>
+    </aside>
+    <section class="flexbox-container">
+      <h2 class="above-flex">Meet Some Doggos</h2>
+      <article class="flex-dog">
+        <img src="images/murphy-card.jpg" alt="light brown boxer mix with big eyes" width="175" onclick="alert('Murphy, Boxer Mix, $123.45')">
+        <h2>Murphy</h2>
+        <h3><b>Cost to Adopt:</b> $123.45</h3>
+        <p>90s ipsum Bill gates world wide web smarter child rollerblades nylon windbreaker, leggings aviators I will always love you wayne gretzky clueless tupac shakur.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+      <article class="flex-dog">
+        <img src="images/poppy-card.jpg" alt="white bichon shih tzu mix running" width="175" onclick="alert('Poppy, Shih Tzu Mix, $123.45')">
+        <h2>Poppy</h2>
+        <h3><b>Cost to Adopt:</b> $123.45</h3>
+        <p>90s ipsum Bill gates world wide web smarter child rollerblades nylon windbreaker, leggings aviators I will always love you wayne gretzky clueless tupac shakur.</p>
+        <button onclick="addToTotal(123.45)">Adopt</button>
+      </article>
+      <article class="flex-dog">
+        <img src="images/jack-card.jpg" alt="white and light brown beagle" width="175" onclick="alert('Jack, Beagle, $123.45')">
```

Comment:
One way you could improve this, is to create a function that calls a function that lives in your JS file, and then that JS function calls the alert. It's a bit overkill for something this simple, but good practice.